contracts/ENSGovernor.sol

```diff
@@ -3,17 +3,18 @@ pragma solidity ^0.8.2;
 
 import "@openzeppelin/contracts/governance/Governor.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorProposalThreshold.sol";
-import "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorVotesQuorumFraction.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol";
 
-contract ENSGovernor is Governor, GovernorProposalThreshold, GovernorCountingSimple, GovernorVotes, GovernorVotesQuorumFraction, GovernorTimelockControl {
+import "./ENSCounting.sol";
+
+contract ENSGovernor is Governor, GovernorProposalThreshold, ENSCounting, GovernorVotes, GovernorVotesQuorumFraction, GovernorTimelockControl {
     constructor(ERC20Votes _token, TimelockController _timelock)
-        Governor("ENS Governor")
-        GovernorVotes(_token)
-        GovernorVotesQuorumFraction(100) // 1%
-        GovernorTimelockControl(_timelock)
+    Governor("ENS Governor")
+    GovernorVotes(_token)
+    GovernorVotesQuorumFraction(100) // 1%
+    GovernorTimelockControl(_timelock)
     {}
 
     function votingDelay() public pure override returns (uint256) {
```

Comment:
A 1 block voting delay is an ill-advised security risk. It reduces the economic exposure a would-be governance attacker has to the results of their proposal (because they can propose and vote and dump the tokens immediately). When multi-block MEV services inevitably become available, it will be possible to do this atomically in one step, essentially attempting to attack the DAO with zero actual economic exposure. I would strongly suggest setting a value of at least a day for the voting delay. Many major DAOs use 2 or 3 days.